app/views/admin/registrations/index.html.haml

```diff
@@ -21,70 +21,96 @@
 .row
   .col-md-12
     %div.margin-event-table
-      %table.datatable#registrations
+      %table.datatable#registrations{ data: { source: admin_conference_registrations_path(conference_id: @conference, format: :json) } }
         %thead
           %tr
-            %th ID#
-            %th Name
-            %th E-Mail
-            - if @code_of_conduct
-              %th
-                %abbr{ title: 'Code of Conduct' } CoC
-            %th Arrival
-            %th Departure
-            - if @conference.questions.any?
-              %th Questions
-            %th Actions
+            %th{ width: '0' } ID#
+            %th{ width: '25%' } Name
+            %th{ width: '0' } Roles
+            %th{ width: '0' } E-Mail
+            %th{ width: '0' }
+              %abbr{ title: 'Code of Conduct' } CoC
+            %th{ width: '0' } Arrival
+            %th{ width: '0' } Departure
+            %th{ width: '0' } Actions
         %tbody
-          - @registrations.each_with_index do |registration, index|
-            %tr
-              %td
-                = registration.id
-              %td
-                = registration.name.present? ? registration.name : registration.username
-                %br
-                - registration.user.roles.where(resource: @conference).each do |role|
-                  %span.label.label-info
-                    = role.name.titleize
-              %td
-                = registration.email
-              - if @code_of_conduct
-                - if registration.accepted_code_of_conduct
-                  %td.text-success.text-center= fa_icon('check', title: 'accepted')
-                - else
-                  %td.text-center.text-warning
-                    = fa_icon('exclamation-circle',
-                      title: 'Has not accepted Code of Conduct')
-              %td
-                - if registration.arrival
-                  = registration.arrival.strftime('%d %b %H:%M')
-                - else
-                  n/a
-              %td
-                - if registration.departure
-                  = registration.departure.strftime('%d %b %H:%M')
-                - else
-                  n/a
-              -if @conference.questions.any?
-                %td
-                  = link_to 'Questions','#', class: 'btn btn-success question-btn', 'data-id' => index, 'data-name' => registration.name
-              %td{ 'data-order' => registration.attended.to_s }
-                = check_box_tag "#{@conference.short_title}_#{registration.id}", registration.id, registration.attended,
-                  class: 'switch-checkbox',
-                  url: toggle_attendance_admin_conference_registration_path(@conference.short_title, id: registration.id)+"?attended="
```

Comment:
@bear454 you have completed removed attendance from the registrations. Why?

Reply:
Because it's useless. It's not an effective sort, or filter, you can't get an overall count of it, and it's only set if you use one of the check-in methods (which are _terrible_ upstream).

Reply:
I understand the concerns, however I don't agree with the removal. I am pretty sure there are some conferences using this feature. 

I didn't expect a PR that aims to improve performance, to actually remove a feature we had :/ 
If this was discussed, I've clearly missed the conversation.